### PR TITLE
Show cluster breadcrumbs

### PR DESCRIPTION
**Pull request by gzdunek**
Closes https://github.com/gravitational/webapps.e/issues/318
image

### CODE REVIEW

File: packages/teleterm/src/ui/services/workspacesService/documentsService/documentsUtils.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    case 'doc.cluster':
+      return document.clusterUri;
+    case 'doc.gateway':
+      return document.targetUri;
+    case 'doc.terminal_tsh_node':
+      return document.serverUri;
+    case 'doc.terminal_tsh_kube':
+      return document.kubeUri;
+    case 'doc.terminal_shell':
+      return routing.getClusterUri(document);
```
Comment (gzdunek) [inline packages/teleterm/src/ui/services/workspacesService/documentsService/documentsUtils.ts:15]: I'm considering adding a `clusterUri` field to `DocumentPtySession` (that field would be constructed in the same way as here, basing on `rootClusterId` and `leafClusterId`).

File: packages/teleterm/src/ui/StatusBar/useActiveDocumentClusterBreadcrumbs.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  const rootCluster = ctx.clustersService.findCluster(rootClusterUri);
+  const leafCluster =
+    clusterUri === rootClusterUri
+      ? undefined
+      : ctx.clustersService.findCluster(clusterUri);
+
+  return [rootCluster, leafCluster]
+    .filter(Boolean)
+    .map(c => c.actualName)
```
Comment (ravicious) [inline packages/teleterm/src/ui/StatusBar/useActiveDocumentClusterBreadcrumbs.ts:34]: Are we sure that `actualName` will always be available here?

File: packages/teleterm/src/ui/services/workspacesService/documentsService/documentsUtils.ts (hunk 1 above)
Comment (ravicious) [reply packages/teleterm/src/ui/services/workspacesService/documentsService/documentsUtils.ts:15]: Good idea, although I'd probably leave this for later. I looked into that a couple of weeks ago and that whole document could use some refactoring if we were to start changing those fields.

**Review by ravicious — APPROVED** [review]

File: packages/teleterm/src/ui/StatusBar/useActiveDocumentClusterBreadcrumbs.ts (hunk 2 above)
Comment (gzdunek) [reply packages/teleterm/src/ui/StatusBar/useActiveDocumentClusterBreadcrumbs.ts:34]: Yes, for leaf clusters we just return cluster name as `actualName`.

**Pull request merged**